SU-HSDO/suhumsci
# SHS-5819:  Remove the Row component from the platform

# READY FOR REVIEW

## Summary
- Remove the `hs_row` paragraph type
- Uninstall `hs_row` module
- Update `stylelint` in `humsci_basic` theme to latest version, to remove deprecation error

**NOTE:** There will likely be merge conflicts after merging #1625, the update hook will need to be updated.

## Need Review By (Date)
10/16

## Urgency
medium

## Steps to Test
- Visit `/admin/structure/paragraphs_type`, confirm that the Row (`hs_row`) paragraph type is not present anymore
- Visit `/admin/modules`, search for `hs_row` and confirm that the "H&S Row Paragraph" module is disabled

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208519242815239
